web/usuaris/src/components/organisms/UsersTable/UsersTable.tsx

```diff
@@ -232,8 +225,8 @@ export const UsersTable: FC<TUsersTable> = ({ filtersSelected }) => {
         </ActionsHeader>
       ),
       cell: ({ row }) => {
-        const status: string = row.getValue('status')
-        const id: string = row.getValue('id')
+        const status: UserStatus = row.getValue('status')
+        const id: UserStatus = row.getValue('id')
```

Comment:
Why the id is changed to UserStatus? I think this id type is string, as it was. It is the user id that has to be passed when calling handleStatus to update the user.

Reply:
You're right, it is just a mistake copying and pasting!